package.json

```diff
@@ -23,11 +23,11 @@
     "watch": "tsc --watch",
     "clean": "rimraf build/",
     "build": "tsc",
-    "dev:load:commonjs": "yarn build; echo 'Loaded Config:\n'; cross-env RUNTIME_ENVIRONMENT=development time node -p -e 'const StrongConfig = require(\".\"); JSON.stringify((new StrongConfig()).load(\"./example\"))' | jq .",
-    "dev:load:es6": "echo 'Loaded Config:\n'; cross-env RUNTIME_ENVIRONMENT=development time ts-node -T -p -e 'import StrongConfig from \".\"; JSON.stringify((new StrongConfig()).load(\"example\"))' | jq .",
+    "dev:load:commonjs": "yarn build; echo 'Loaded Config:\n'; cross-env NODE_ENV=development time node scripts/dev/load-commonjs.js | jq .",
+    "dev:load:es6": "echo 'Loaded Config:\n'; cross-env NODE_ENV=development time ts-node -T scripts/dev/load-es6.ts | jq .",
```

Comment:
Small improvement suggestion to get rid of the `echo` and the `jq` dependency here: (check the corresponding suggestion in `scripts/load-es6.ts`

```suggestion
    "dev:load:es6": "cross-env NODE_ENV=development time ts-node --transpile-only scripts/dev/load-es6.ts",
```

Same comment would apply to the other yarn tasks here, just wanted to align on the general approach first

Reply:
I tested this but it seems to show a general `console.log` type of output, which is why I went for `| jq .`. Can't see a difference with `console.dir` and `{colors: true}`. `jq` does a much better job in color-coding and indentation.

Also, `jq` would be just a dependency for developers, imo acceptable

Reply:
interesting. how did you test this? here's a comparison screenshot from iTerm with my suggestion solution on the left, and the current jq solution on the right:

<img width="1576" alt="Screenshot 2019-11-07 at 16 52 30" src="https://user-images.githubusercontent.com/117495/68404969-9a55a580-017f-11ea-8833-8023a5dab13d.png">

There's another alternative to `console.dir()` as well:

```js
import { inspect } from 'util'
console.log(inspect(config, { colors: true, compact: false }))
```

which looks like this (again new solution left, old one right)

<img width="1575" alt="Screenshot 2019-11-07 at 16 58 28" src="https://user-images.githubusercontent.com/117495/68405141-df79d780-017f-11ea-8d15-3fde9bf095b8.png">

Reply:
alright, `inspect` then. I would have preferred 4 chars much more: `| jq .`

It's updated.

Reply:
thanks, IMO requiring an additional dependency just for the blue color is worse for users in the end. plus more to write in the README ;)